psd-web/app/decorators/investigation_decorator.rb

```diff
@@ -38,13 +38,18 @@ def product_summary_list
     h.render "components/govuk_summary_list", rows: rows, classes: "govuk-summary-list--no-border"
   end
 
-  def display_risk_and_issues_list?
-    object.hazard_type.present? || object.non_compliant_reason.present?
+  def risk_level
+    object.risk_level.to_s
   end
```

Comment:
suggestion: rename this as something like `risk_level_description` and move the logic to return "Not set" if null here?

```suggestion
  def risk_level_description
    object.risk_level || "Not set"
  end
```